articles/libraries/auth0js/migration-guide.md

```diff
@@ -0,0 +1,62 @@
+---
+section: libraries
+description: How to migrate from auth0.js v7 to auth0.js v8
+url: /libraries/auth0js/migration-guide
+---
+
+# Auth0.js v7 to v8 Migration Guide
+
+The following instructions assume you are migrating from **auth0.js v7** to **auth0.js v8**. 
+
+The goal of this migration guide is to provide you with all of the information you would need to update Auth0.js in your application. Of course, your first step is to include the latest version of auth0.js. Beyond that, take a careful look at each of the areas on this page. You will need to change your implementation of auth0.js to reflect the new changes. Take a look below for more information!
+
+## Initialization of auth0.js
+
+Initialization of auth0.js in your application will now use `auth0.WebAuth` instead of `Auth0`
+```html
+<script src="https://cdn.auth0.com/js/auth0/8.0.4/auth0.min.js"></script>
+<script type="text/javascript">
+  var webAuth = new auth0.WebAuth({
+    domain:       'YOUR_AUTH0_DOMAIN',
+    clientID:     'YOUR_CLIENT_ID'
+  });
+</script>
+```
+
+## Login
```

Comment:
Think the actual reference doc should explain all of these scenarios too, probably? Right now it only shows a couple of them. I can walk through the different scenarios and show them without much trouble I think. Might add that into this PR and have you review.